packages/cli/src/eventbus/message-event-bus-destination/message-event-bus-destination.ee.ts

```diff
@@ -40,6 +44,21 @@ export abstract class MessageEventBusDestination implements MessageEventBusDesti
 		this.logger = Container.get(Logger);
 		this.license = Container.get(License);
 
+		const maxDuration = options.circuitBreaker?.maxDuration ?? 3 * Time.minutes.toMilliseconds;
+		const maxFailures = options.circuitBreaker?.maxFailures ?? 5;
+		const halfOpenRequests = options.circuitBreaker?.halfOpenRequests ?? 2;
+		const failureWindow = options.circuitBreaker?.failureWindow ?? 1 * Time.minutes.toMilliseconds;
+		const maxConcurrentHalfOpenRequests =
+			options.circuitBreaker?.maxConcurrentHalfOpenRequests ?? 1;
```

Comment:
Similar suggestion for constants.

```suggestion
const DEFAULT_MAX_DURATION_MS = 3 * Time.minutes.toMilliseconds;
const DEFAULT_MAX_FAILURES = 5;
const DEFAULT_HALF_OPEN_REQUESTS = 2;
const DEFAULT_FAILURE_WINDOW_MS = 1 * Time.minutes.toMilliseconds;
const DEFAULT_MAX_CONCURRENT_HALF_OPEN_REQUESTS = 1;

const maxDuration = options.circuitBreaker?.maxDuration ?? DEFAULT_MAX_DURATION_MS;
const maxFailures = options.circuitBreaker?.maxFailures ?? DEFAULT_MAX_FAILURES;
const halfOpenRequests = options.circuitBreaker?.halfOpenRequests ?? DEFAULT_HALF_OPEN_REQUESTS;
const failureWindow = options.circuitBreaker?.failureWindow ?? DEFAULT_FAILURE_WINDOW_MS;
const maxConcurrentHalfOpenRequests =
  options.circuitBreaker?.maxConcurrentHalfOpenRequests ?? DEFAULT_MAX_CONCURRENT_HALF_OPEN_REQUESTS;
```